src/components/autocomplete/Autocomplete.vue

```diff
@@ -38,9 +38,16 @@
                     class="dropdown-content"
                     v-show="isActive"
                     :style="contentStyle">
-                    <div v-if="hasHeaderSlot" class="dropdown-item">
+                    <a
+                        v-if="hasHeaderSlot"
+                        class="dropdown-item"
+                        role="button"
+                        tabindex="0"
+                        :class="{ 'is-hovered': 'header' === hovered }"
```

Comment:
What happen if an option value is "header" or "footer" ?

Reply:
Yes, this would be an issue. I am resolving this now by using an internal variable for managing the state of hover for the header/footer and leaving the hovered variable for just items in the list.